packages/driver/test/cypress/integration/commands/actions/select_spec.coffee

```diff
@@ -79,6 +79,22 @@ describe "src/cy/commands/actions/select", ->
       cy.get("select[name=movies]").select(["apoc", "br"]).then ($select) ->
         expect($select.val()).to.deep.eq ["apoc", "br"]
 
+    it "clears previous values when providing an empty array", ->
+      ## make sure we have previous values
+      select = cy.$$("select[name=movies]").val(["br", "2001"])
+      expect(select.val()).to.deep.eq ["br", "2001"]
+
+      cy.get("select[name=movies]").select([]).then ($select) ->
+        expect($select.val()).to.deep.eq []
+
+    it "clears previous value when providing an empty string to a single select", ->
+      ## make sure we have a previous value
+      select = cy.$$("select[name=foods]").val("Ramen")
+      expect(select.val()).to.eq "Ramen"
+
+      cy.get("select[name=foods]").select("").then ($select) ->
+        expect($select.val()).to.eq null
+
```

Comment:
it might be worth adding a couple extra tests that doing `cy.select("")` or `cy.select([])` on an already-unselected `<select>` doesn't cause a crash or modify the selected elements, since it's an edge case

Reply:
Good idea, I'll add a few more to cover edge cases.